### ISSUE TITLE
Environment Version

### ISSUE THREAD

**Issue by yeah334** [dvlab-research/Stratified-Transformer#34]
I have a problem with the installation of pyg.
Here is my environment: python=3.7, pytorch=1.8.0, cudatoolkit=11.1, other packages are default version.

Console returns:
''
Package libstdcxx-ng conflicts for:
pytorch==1.8.0 -> cudatoolkit[version='>=11.1,<11.2'] -> libstdcxx-ng[version='>=10.3.0|>=7.5.0|>=7.3.0|>=7.2.0|>=11.2.0']
gmp -> libstdcxx-ng[version='>=7.2.0|>=7.3.0|>=7.5.0']
libtiff -> libstdcxx-ng[version='>=4.9|>=7.3.0|>=7.5.0|>=7.2.0']
torchvision==0.9.0 -> cudatoolkit[version='>=11.1,<11.2'] -> libstdcxx-ng[version='>=10.3.0|>=7.3.0|>=7.5.0|>=11.2.0|>=8.4.0|>=7.2.0']

....

  - pytorch==1.8.0 -> cudatoolkit[version='>=11.1,<11.2'] -> __glibc[version='>=2.17,<3.0.a0']
  - readline -> libgcc-ng[version='>=7.5.0'] -> __glibc[version='>=2.17']
  - scipy -> libgcc-ng[version='>=7.5.0'] -> __glibc[version='>=2.17']
  - sqlite -> libgcc-ng[version='>=7.5.0'] -> __glibc[version='>=2.17']
  - tk -> libgcc-ng[version='>=7.5.0'] -> __glibc[version='>=2.17']
  - torchvision==0.9.0 -> cudatoolkit[version='>=11.1,<11.2'] -> __glibc[version='>=2.17,<3.0.a0']
  - xz -> libgcc-ng[version='>=7.5.0'] -> __glibc[version='>=2.17']
  - zlib -> libgcc-ng[version='>=7.5.0'] -> __glibc[version='>=2.17']

Your installed version is: 2.31
''
Can you list the specific versions of your environments? Thanks very much.

**Comment by X-Lai** [comment]
I have updated the guide of environment installation. Sorry for the inconvenience and try again. If you have any further problem, please help to report it.

**Comment by yeah334** [comment]
> 

No. My ubuntu is 20.04 which is only support cuda 11.0 and above.

**Comment by L-Reichardt** [comment]
@yeah334 Ive managed to get it to run on 20.04 LTS with CUDA 11.3. However I installed everything with pip making sure I check the original source for the correct version.

I've also compiled every necessary package outside of anaconda, then manually copied the installed files into the corresponding Anaconda environment.

I use :
- pytorch==1.11.0+cu11.3
- torch_sparse==0.6.13
- torch_points3d==1.3.0

**Comment by yeah334** [comment]
@L-Reichardt Nice! I will try this way if I need to run this project later.
